File: trail1/알파벳에 따른 평가/evaluation-by-alphabet.cs
using System;
using System.Collections.Generic;

public class Codetree
{  
    public static void Main()
    {
        var Score = new Dictionary<string, string> {
            ["S"] = "Superior", ["A"] = "Excellent", ["B"] = "Good", ["C"] = "Usually", ["D"] = "Effort"
        };
        string grade = Console.ReadLine();
        if (Score.TryGetValue(grade, out string result)) {
            Console.WriteLine(result);
        } else {Console.WriteLine("Failure");}
    }
}

File: trail1/알파벳에 따른 평가/evaluation-by-alphabet.java
import java.util.Scanner;
import java.util.HashMap;
import java.util.Map;
public class Main {
    public static void main(String[] args) {
        Map<String, String> score = new HashMap<>();
            score.put("S", "Superior");
            score.put("A", "Excellent");
            score.put("B", "Good");
            score.put("C", "Usually");
            score.put("D", "Effort");
        
        try (Scanner sc = new Scanner(System.in)) {
            if (sc.hasNext()) {
                String grade = sc.next();
                String result = score.getOrDefault(grade, "Failure");
                System.out.println(result);
            }
        }
    }
}